internal/fs/wrappers/monitoring.go

```diff
@@ -42,7 +54,20 @@ func init() {
 			Aggregation: view.Sum(),
 			TagKeys:     []tag.Key{tags.FSOp, tags.FSError},
 		},
-	); err != nil {
+		&view.View{
+			Name:        "fs/ops_error_count",
+			Measure:     opErrorCount,
+			Description: "The cumulative number of errors generated by file system operations",
+			Aggregation: view.Sum(),
+			TagKeys:     []tag.Key{tags.FSError},
+		},
+		&view.View{
+			Name:        "fs/ops_latency",
```

Comment:
use same name for Name and measure attributes i.,e ops_latency -> opsLatency. Either call it ops or op in both places. Same for error metric